src/main/java/com/softserve/borda/controllers/AuthController.java

```diff
@@ -27,15 +28,15 @@ public AuthController(UserService userService, JwtProvider jwtProvider, Password
     }
 
     @PostMapping("/register")
-    public String registerUser(@RequestBody @Valid RegistrationRequest registrationRequest) {
+    public RedirectView registerUser(RegistrationRequest registrationRequest) {
         User user = new User();
         user.setUsername(registrationRequest.getLogin());
         user.setPassword(passwordEncoder.encode(registrationRequest.getPassword()));
         user.setEmail(registrationRequest.getEmail());
         user.setFirstName(registrationRequest.getFirstName());
         user.setLastName(registrationRequest.getLastName());
         userService.createOrUpdate(user);
-        return "OK";
+        return new RedirectView("/pages/personal-account.html");
```

Comment:
Rest API does not know about `/pages/personal-account.html` page. Rest Controller should return http status code and in meantime, UI should handle http status code and decide what to do.
For example:
(Negative scenario) A user tries to log in and filled the wrong password then back-end returns HTTP status code 400 (Bad request).
(Positive scenario) A user tries to log in and filled the correct credentials then back-end returns HTTP status code 200 (OK) and a bearer token exists in the response body.